chore: reuse templates

Roming22 opened the pull request:
cf [RHTAPINST-50](https://issues.redhat.com//browse/RHTAPINST-50)

Link identical template files to a single instance: service-account.

rh-pre-commit.version: 2.3.2
rh-pre-commit.check-secrets: ENABLED

otaviof reviewed (approved):
Nice one, @Roming22! Thanks.

/lgtm

Roming22 commented:
/retest

Roming22 commented:
/retest

Roming22 commented:
/retest

Roming22 commented:
/retest

Roming22 commented:
/retest

Roming22 commented:
/retest

xinredhat commented:
/retest

xinredhat commented:
it failed at rhtap-install task and reported error 
```
Error: install failed: template: rhtap-acs-test/templates/service-account.yaml:2:4: executing "rhtap-acs-test/templates/service-account.yaml" at <include "common.serviceAccount" .>: error calling include: template: no template "common.serviceAccount" associated with template "gotpl"

```

Roming22 commented:
/retest

Roming22 commented:
/retest

Roming22 commented:
/retest

Roming22 committed:
chore: reuse templates

cf RHTAPINST-50

Link identical template files to a single instance: service-account.

rh-pre-commit.version: 2.3.2
rh-pre-commit.check-secrets: ENABLED

Roming22 commented:
/retest

Roming22 commented:
Force merging as CI is unstable and tests were successful on 4.17.
